src/containers/guest-home-page/login-form/LoginForm.tsx

```diff
@@ -35,26 +52,41 @@ const LoginForm = ({
     openModal({ component: <ForgotPassword /> })
   }
 
+  const handleCheckboxChange = (
```

Comment:
Is it necessary to create a new function `handleCheckboxChange` instead of using `handleChange`?